### ISSUE TITLE
Error while training the tensorflow model 

### ISSUE THREAD

**Issue by avinash8494** [MuhammedBuyukkinaci/TensorFlow-Sentiment-Analysis-on-Amazon-Reviews-Data#2]
All of the training underwent good on GPU Machine, but the training has been stopped before the training steps like for 18000 steps it is showing Training has finished. What was the issue?

**Comment by MuhammedBuyukkinaci** [comment]
Hi,
Thanks for your great question.
I wrote a customized early stopping function (early_stopping_check) meaning that if loss on test data didn't decrease for some time period, the models stops.
Please like the repository for more repositories and contributions.
